### PR TITLE
Adding about external_labels in katacoda tutorial

### LINKED ISSUE #2690: Add about external_labels in katacoda tutorial
```global:
  scrape_interval: 15s
  evaluation_interval: 15s
  external_labels:
    cluster: eu1
    replica: 0
```

In the tutorial, https://katacoda.com/thanos/courses/thanos/1-globalview, it is important to add more information (note) about commands.

### PR DESCRIPTION
**Pull request by soniasingla**
Signed-off-by: soniasingla <soniasingla.1812@gmail.com>

* [ ] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Fixes #2690 

CC : @povilasv @bwplotka

### CODE REVIEW

**Comment by yeya24** [comment]
CI is broken. Would you mind rebasing master?

**Comment by soniasingla** [comment]
hey @yeya24 ! it's ready for another review 🧐

File: tutorials/katacoda/thanos/1-globalview/step1.md [hunk 1]
```diff
@@ -70,6 +70,8 @@ scrape_configs:
       - targets: ['127.0.0.1:9091','127.0.0.1:9092']
 </pre>
 
+**NOTE** : The `global` entry should have `external_labels` else Thanos will not recognise it.
```
Comment (bwplotka) [inline tutorials/katacoda/thanos/1-globalview/step1.md:73]: I think we can be more helpful and this and describe it more. What do you think?

Think about someone new here. What it means `recognise` here? What if it's empty? What can go wrong? :hugs:

Comment (povilasv) [reply tutorials/katacoda/thanos/1-globalview/step1.md:73]: :+1: I think let's describe why do we need this, what this does, the effects of deduplication.

File: tutorials/katacoda/thanos/1-globalview/step1.md [hunk 2]
```diff
@@ -70,6 +70,8 @@ scrape_configs:
       - targets: ['127.0.0.1:9091','127.0.0.1:9092']
 </pre>
 
+**NOTE** : Every Prometheus instance must have a globally unique set of identifying labels. These labels are used by the sidecar, Querier to filter out store APIs, and various object readers. It is important to provide compatable set of external labels as Thanos aggregate data across all the available instances.
```
Comment (bwplotka) [inline tutorials/katacoda/thanos/1-globalview/step1.md:73]:
```suggestion
**NOTE** : Every Prometheus instance must have a globally unique set of identifying labels. These labels are important as they represent certain "stream" of data (e.g in the form of TSDB blocks). Within those exact external labels, the compactions and downsampling are performed, Querier filters its store APIs, further sharding option, deduplication, and potential multi-tenancy capabilities are available. Those are not easy to edit retroactively, so it's important to provide a compatible set of external labels as in order to for Thanos to aggregate data across all the available instances. 
```

Comment (povilasv) [reply tutorials/katacoda/thanos/1-globalview/step1.md:73]: :+1:

**Comment by soniasingla** [comment]
> Nice, potential suggestion to add even more context. What do you think? 🤗

💪🏻

**Comment by soniasingla** [comment]
@bwplotka it's ready :)

**Comment by soniasingla** [comment]
@bwplotka idk its weird but without pushing these auto-generated files , CI is breaking ,saying you have uncommitted changes , I re-based the branch on master as welll 🙁

**Comment by bwplotka** [comment]
I think you definitely commited too many files, we don't need those flags.txt ): Somehow you have `txt-e` at the end of it :thinking:

File: docs/components/flags/compact.txt-e [hunk 3]
```diff
@@ -0,0 +1,114 @@
+usage: thanos compact [<flags>]
+
```
Comment (bwplotka) [inline docs/components/flags/compact.txt-e:2]: We don't need to commit those ): Should be git ignored, but I guess you "escaped" gitignore with `-e` file suffix.

**Comment by Harshitha1234** [comment]
Maybe try running the below command and push again 🙂 
`./scripts/cleanup-white-noise.sh tutorials/katacoda/thanos/1-globalview/step1.md`

**Comment by soniasingla** [comment]
Thanks @Harshitha1234, the error was related to Mac sed command issues 🍨

**Pull request merged**